compiler/src/dotty/tools/dotc/printing/UserFacingPrinter.scala

```diff
@@ -0,0 +1,71 @@
+package dotty.tools.dotc
+package printing
+
+import core._
+import Constants.Constant, Contexts.Context, Denotations._, Flags._, Names._
+import NameOps._, StdNames._, Decorators._, Scopes.Scope, Types._, Texts._
+import SymDenotations.NoDenotation, Symbols.{ Symbol, ClassSymbol, defn }
+
+class UserFacingPrinter(_ctx: Context) extends RefinedPrinter(_ctx) {
```

Comment:
Do you use `_ctx` because `ctx` is already defined in `RefinedPrinter`?

Reply:
Yes and that's what we do in RefinedPrinter when extending plain printer :)